software/firmware/europi.py

```diff
@@ -44,6 +45,8 @@
 CHAR_WIDTH = 8
 CHAR_HEIGHT = 8
 
+# This env var is set when we don't have any real hardware, for example in a test or doc generation setting
+TEST_ENV = os.getenv('TEST_ENV', 0)

```

Comment:
Does `os.getenv()` work on the pico hardware? I assume it would always just return 0 since there's no os to set env vars. I want to make sure it doesn't throw an error.

Reply:
fixed